src/test/scala/de/frosner/dds/core/DDSTest.scala

```diff
@@ -251,6 +251,16 @@ class DDSTest extends FlatSpec with Matchers with MockFactory with BeforeAndAfte
     actualChart.frequencies.toList shouldBe List(3,1,2)
   }
 
+  it should "be served from a single numeric value RDD, binned according to Sturge's formula" in {
+    DDS.start(mockedServer)
+    val values = sc.makeRDD(List(0,5,15,3,8))
+    DDS.histogram(values)
+
+    val actualChart = mockedServer.lastServed.get.asInstanceOf[Histogram]
+    actualChart.bins.toList shouldBe List(0,3.75,7.5,11.25,15.0)
+    actualChart.frequencies.toList shouldBe List(2,1,1,1)
+  }
+
   "Correct pie chart from RDD after groupBy" should "be served when values are already grouped" in {
```

Comment:
@RPCMoritz We need to add a test for an illegal number of buckets (< 2) as well. I will do it once I finished merging.